c_src/ep_encoder.c

```diff
@@ -471,6 +471,63 @@ pack_bytes(ErlNifEnv *env, ERL_NIF_TERM term, ep_enc_t *enc)
     return RET_OK;
 }
 
+static inline int64_t
+to_integer(unsigned char *str, size_t size) {
```

Comment:
http://www.cplusplus.com/reference/cstdlib/atol/

Reply:
i know they exist in cpp. but they dont work with my gcc compiler.. so implemented them myself.

Reply:
atol should work, it's in the stdlib.h header file.

Reply:
yeah, that works - thank you!